components/Order/model_interfaces.rst

```diff
@@ -0,0 +1,84 @@
+Model interfaces
+================
+
+.. _component_order_model_adjustment-interface:
+
+AdjustmentInterface
+-------------------
+This interface provides you with basic management of an adjustment's properties.
+
+.. note::
+    This interface extends the :ref:`component_resource_model_timestampable-interface` and the :ref:`component_originator_model_origin-aware-interface` |br|
+    For more detailed information about the interface go to `Sylius API's AdjustmentInterface`_.
+.. _Sylius API's AdjustmentInterface: http://api.sylius.org/Sylius/Component/Order/Model/AdjustmentInterface.html
+
+.. _component_order_model_adjustable-interface:
+
+AdjustableInterface
+-------------------
+This interface provides basic tools for adjustment management.
+Use this interface to any model that you want to be adjustable.
```

Comment:
Maybe "Use this interface if you want to make some model adjustable" ? Or something other.